### ISSUE TITLE
JsonWebTokenError: jwt malformed since v1.3.0

### ISSUE THREAD

**Issue by philpep** [deeplay-io/verdaccio-openid-connect#9]
Hi,

We use verdaccio-openid-connect with keycloak and authentication broke with version v1.3.0 I guess due to #8 

Use case to reproduce:

```
% npm login
npm notice Log in on http://verdaccio.localhost
Logged in on http://verdaccio.localhost

% cat ~/.npmrc
registry=http://verdaccio.localhost/
//verdaccio.localhost/:_authToken="Rv7+MD8qbJZRA+Bjiyz50NyWmlnKt7L7h2HmH6YCKi0="


% npm show somepackage version
npm ERR! code E401                                                                             
npm ERR! Unable to authenticate, your authentication token seems to be invalid.
npm ERR! To correct this please trying logging in again with:      
npm ERR!     npm login                                                                         
                                                                                               
npm ERR! A complete log of this run can be found in:         
npm ERR!     /home/.../.npm/_logs/2022-05-02T14_44_30_468Z-debug-0.log

```

Server logs are:

```
JsonWebTokenError: jwt malformed                                                                                                                                                              
    at Object.module.exports [as verify] (/usr/lib/node_modules/verdaccio-openid-connect/node_modules/jsonwebtoken/verify.js:63:17)
    at /usr/lib/node_modules/verdaccio-openid-connect/lib/index.js:186:34                                                                                                                     
    at Layer.handle [as handle_request] (/usr/lib/node_modules/verdaccio/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/usr/lib/node_modules/verdaccio/node_modules/express/lib/router/index.js:317:13)                                                                                          
    at /usr/lib/node_modules/verdaccio/node_modules/express/lib/router/index.js:284:7                                                                                                         
    at Function.process_params (/usr/lib/node_modules/verdaccio/node_modules/express/lib/router/index.js:335:12)                                                                              
    at next (/usr/lib/node_modules/verdaccio/node_modules/express/lib/router/index.js:275:10)                                                                                                     at Function.handle (/usr/lib/node_modules/verdaccio/node_modules/express/lib/router/index.js:174:3)
    at router (/usr/lib/node_modules/verdaccio/node_modules/express/lib/router/index.js:47:12)                                                                                                    at Layer.handle [as handle_request] (/usr/lib/node_modules/verdaccio/node_modules/express/lib/router/layer.js:95:5)                                                                       
 http <-- 401, user: null(192.168.42.1 via 10.42.134.162), req: 'GET /tslib/-/tslib-1.14.1.tgz', error: authorization required to access package tslib                                        
 error--- erro while verify jwt bearer token: jwt malformed                                                                                                                                   
```

rolling back to version 1.2.0 fixes the issue.

Thanks!

**Comment by aikoven** [comment]
Cc. @dafanasiev

**Comment by dafanasiev** [comment]
@philpep can you show you config ?

**Comment by dafanasiev** [comment]
I fix legacy mode in #10

**Comment by aikoven** [comment]
Published a new version with the fix: `verdaccio-openid-connect@1.3.1`
